terragrunt/env/common/common_variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "account_id" {
+  description = "(Required) The account ID to perform actions on."
+  type        = string
+}
+
+variable "aws_org_id" {
+  description = "(Required) The AWS org account ID.  Used to limit which roles the API can assume."
+  type        = string
+  sensitive   = true
+}
```

Comment:
I don't think we'll need this var for the project.   We've used it on other projects to control cross-account role assuming (e.g. can account A assume a role in account B).